.env.example

```diff
@@ -1,5 +1,5 @@
 # Shared Environment
-LOCAL_SEED_ADDRESS=0xYOUR_DEV_ADDRESS
+LOCAL_SEED_ADDRESS=0x4156fBD36457Df033f63Cfe14e0D246c1D93ba69
```

Comment:
you can make these changes to your local `.env` and leave this example file generic